open-company/open-company-web
# Mark unread

#### Review after #1081 

Related PRs:
- ~[Storage](https://github.com/open-company/open-company-storage/pull/230)~
- ~[Change](https://github.com/open-company/open-company-change/pull/25)~

Card: https://trello.com/c/MymRMgXo

To test:
- test mark as read on desktop
- test mark as unread on desktop
- test mark as read on mobile
- test mark as unread on mobile
- make sure mark as read remove the post from Unread
- make sure the post comes back when a new comment is added (from another user)
- make sure mark as unread move a post back into Unread
